task/transcode.go

```diff
@@ -1,342 +1,13 @@
 package task
 
 import (
-	"bytes"
-	"context"
-	"errors"
-	"fmt"
-	"io"
 	"math/rand"
-	"os"
-	"sync/atomic"
 	"time"
 
-	"github.com/golang/glog"
-	api "github.com/livepeer/go-api-client"
-	"github.com/livepeer/joy4/av"
-	"github.com/livepeer/joy4/av/avutil"
 	"github.com/livepeer/joy4/format"
-	"github.com/livepeer/joy4/format/mp4"
-	"github.com/livepeer/joy4/format/ts"
-	"github.com/livepeer/livepeer-data/pkg/data"
-	"github.com/livepeer/stream-tester/model"
-	"github.com/livepeer/stream-tester/segmenter"
-)
-
-const (
-	segLen               = 2 * time.Second
-	maxFileSizeForMemory = 50_000_000
 )
 
 func init() {
 	format.RegisterAll()
```

Comment:
Are we using joy4 anywhere else? If not we should be able to remove that dependency as well!!

Reply:
yeah i can't see it being used, removed